nextflow_schema.json

```diff
@@ -0,0 +1,192 @@
+            "description": "Options on minimum values for filtering out input data",
+            "default": "",
+            "properties": {
+                "kraken2_db": {
+                    "type": "string",
+                    "default": "s3://genome-idx/kraken/standard_08gb_20240904",
+                    "fa_icon": "fas fa-server",
+                    "description": "Kraken2 database to use for abundance checks"
+                },
+                "min_abundance_percent": {
+                    "type": "number",
+                    "default": 10,
+                    "description": "Minimum L.pn abundance to keep sample in pipeline",
+                    "fa_icon": "fas fa-percentage",
+                    "minimum": 1,
+                    "maximum": 99
+                },
+                "min_reads": {
+                    "type": "integer",
+                    "default": 150000,
+                    "description": "Minimum number of paired reads to keep sample in pipeline",
+                    "minimum": 0
+                }
+            },
+            "fa_icon": "fas fa-filter"
+        },
+        "skipping_options": {
+            "title": "Skipping options",
+            "type": "object",
+            "description": "Processes that can be skipped through parameters",
+            "default": "",
+            "properties": {
+                "skip_el_gato": {
+                    "type": "boolean",
+                    "description": "Skip running all el_gato Sequence Typing aspects",
+                    "fa_icon": "fas fa-forward"
+                },
+                "skip_plotting": {
+                    "type": "boolean",
+                    "description": "Skip plotting el_gato allele profiles",
+                    "fa_icon": "fas fa-forward"
+                }
+            },
+            "fa_icon": "fas fa-forward"
+        },
+        "quast_options": {
+            "title": "Quast options",
+            "type": "object",
+            "description": "Options for quast",
+            "default": "",
+            "properties": {
+                "quast_ref": {
+                    "type": "string",
+                    "default": "data/C9_S.reference.fna",
+                    "format": "file-path",
+                    "fa_icon": "fas fa-file-alt",
+                    "description": "Path to reference fasta file to base quast metrics off of"
+                }
+            }
+        },
+        "chewbbaca_options": {
+            "title": "Chewbbaca options",
+            "type": "object",
+            "description": "Options to specify the chewbbaca schema to use",
+            "default": "",
+            "properties": {
+                "schema_targets": {
+                    "type": "string",
+                    "description": "Schema targets"
+                },
+                "prepped_schema": {
+                    "type": "string",
+                    "default": "data/SeqSphere_1521_schema",
```

Comment:
This can be for later, but I would recommend a bit of change in logic here for integration with IRIDA Next. Specifically, I'd recommend leaving this to a default of empty and in the pipeline, if empty, using the default `data/SeqSphere_1521_schema` that comes with the pipeline code.

That way, in IRIDA Next, we can always override it later on if it needs to be updated, but otherwise will default to the database distributed with the pipeline.